site/src/_pages/docs/0_index.mdx

```diff
@@ -9,6 +9,20 @@ The following pages provide a practical introduction for developers who want to:
 
 For the full technical details, read [the Block Protocol specification](https://blockprotocol.org/spec).
 
+### Why is structured data useful?
+
+Structured data allows computers to understand information. It is easier to create and easier to understand than unstructured data.
+
+Most of the data on the web is unstructured. When we write webpages in HTML, it's easily readable by humans, but not by computers. The computer only knows you've added a bunch of `<p>` tags to a page, but doesn't know anything about what you've written inside them.
+
+Unlike plain HTML, blocks created with the Block Protocol have **types**, such as _Person_, _Company_, and _Date_ that help computers understand what your text is about.
+
+{diagram of HTML vs. structure block data}
```

Comment:
@MaggieAppleton We should ready this for close of play Wednesday or remove this placeholder from the docs. Do you think we'll have something for then?